plugins/generic/pln/classes/DepositPackage.inc.php

```diff
@@ -111,19 +132,19 @@ function generateAtomDocument() {
 		$entry->setAttributeNS('http://www.w3.org/2000/xmlns/' ,'xmlns:dcterms', 'http://purl.org/dc/terms/');
 		$entry->setAttributeNS('http://www.w3.org/2000/xmlns/' ,'xmlns:pkp', 'http://pkp.sfu.ca/SWORD');
 		
-		$email = $atom->createElement('email', $journal->getSetting('contactEmail'));
+		$email = $this->_generateElement($atom, 'email', $journal->getSetting('contactEmail'));
```

Comment:
Some of these fields should definitely not contain entities -- fields like the contact email are displayed client-side using the Smarty `|escape` filter, which means that `&copy;` will be dumped out to the browser escaped. Generally entities would only come in via TinyMCE-enabled fields.